src/lib/components/FormModel/index.js

```diff
@@ -1,5 +1,3 @@
-import FormModel from './live-form-model.component';
-import AutoSaveDefaultSpinner from './children/autosave-default-spinner.component';
+import { FormModel } from './form-model.component';
```

Comment:
Why was the AutoSaveSpinner deleted from here? It was not strictly part of the form model, but more of an example of how to make autosave spinners (which are dynamically added to the form)

Reply:
Re-added a spinner (which now that I think it should be more of a generic indicator) with a caveat. The previous implementation only worked if `autosave` was passed down to the form, if not then there was no indicator at all. Now we have a single indicator for the whole form so it is not next to each field but at the beginning.

It is possible to improve this by updating `formActions.saveData()` to report back which fields were successful and which were not.

Reply:
A couple of notes we discussed offline about the autosavespinner.

First, there are two different save workflows, the first is a full form save, and the second workflow is autosave. This autosave component is intended to only show when autosave is enabled on the form, and is a component per-field that executes on field save. The example default component that ships with the component library is purposely typography-free and not intended for use (I believe it shows text of "Saving..." and "success" or some such). It is intended to be passed in from the client application and rendered that way, as demonstrated in the generator application.